src/App.jsx

```diff
@@ -1,25 +1,27 @@
 import React from 'react';
-import logo from './logo.svg';
-import './App.css';
+import { BrowserRouter as Router, Routes, Route } from 'react-router-dom';
+import './index.css';
+import Dashboard from './pages/Dashboard/Dashboard';
+import Profile from './pages/Profile/Profile';
+import Surveys from './pages/Surveys/Surveys';
+import Data from './pages/Data/Data';
+import ImportCsv from './pages/ImportCsv/ImportCsv';
+import AddData from './pages/Data/Add/AddData';
 
 function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
+    <React.StrictMode>
+      <Router>
+        <Routes>
```

Comment:
even though it's not defined in the proj requirements doc yet we should add a route for `/login` since auth is one of the next priorities for this project; we'll need a page where we can do login/register functionality and it's better just to set it up now with the rest of the routes on this pr